### DISCUSSION TITLE
Two computers use the same account number

### DISCUSSION THREAD

**Discussion by w1511025** [Chia-Network/chia-blockchain#6227] [Support]
Two computers use the same account number, why does not the total number of farmland display (the node has been synchronized)

**Comment by BrandtH22** [comment]
@w1511025 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - you would need to setup the second machine to properly connect to the first in order for the farming space to be identified https://docs.chia.net/farming-on-many-machines/

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
